utopia-remix/app/routes/projects.tsx

```diff
@@ -207,6 +207,26 @@ const ProjectsPageInner = React.memo(() => {
     return activeProjects.find((p) => p.proj_id === sharingProjectId) ?? null
   }, [activeProjects, sharingProjectId])
 
+  const setSelectedProjectId = useProjectsStore((store) => store.setSelectedProjectId)
+
+  const containerRef = React.useRef<HTMLDivElement | null>(null)
+
+  const onContainerMouseDown = React.useCallback(
+    (e: React.MouseEvent) => {
+      const target = e.target as HTMLElement
+      // check if target is inside the projects' container
+      // since this handler fires also on context menu clicks
```

Comment:
I believe you can exclude context menu clicks by checking `e.type !== 'contextmenu'` BTW if that would change the logic.

Reply:
Yes, but maybe in the future we will also open it from a button, etc, so I just wanted to give the proper check here (that the clicked element is indeed inside the projects page container)